### ISSUE TITLE
BTNPIN flag does not work for multiple buttons

### ISSUE THREAD

**Issue by ZOUDIO** [Aircoookie/WLED#3792]
### What happened?

The BTNPIN flag does not seem to be working for multiple pins. Only the first pin gets added.

### To Reproduce Bug

I used the buildflag '-D BTNPIN=14,32' but only 14 shows up in the GUI
![button](https://github.com/Aircoookie/WLED/assets/48258407/c97f8a6f-c260-4378-9a7c-588dae4c92f6)

### Expected Behavior

I would expect multiple buttons to be initialized and visible in the web GUI.

### Install Method

Self-Compiled

### What version of WLED?

v0.14.1

### Which microcontroller/board are you seeing the problem on?

ESP32

### Relevant log/trace output

```shell
-
```

### Anything else?

I used the same type of buildflag for multiple LED output pins, and this is working:
-D DATA_PINS=2,4,12,13,15,17

So it is specifically related to this build flag it seems

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blazoncek** [comment]
Available in 0.15.0 beta 1.

**Comment by Arcitec** [comment]
@ZOUDIO Your example is missing `BTNTYPE`. You must set that too, if you want to pre-compile multiple pins. Otherwise the other button pins will default to "disabled".

Check the source code for values, but the most common value is `2` (Pushbutton), so in your case you would have to do:

```ini
  -D WLED_MAX_BUTTONS=2
  -D BTNPIN=14,32
  -D BTNTYPE=2,2
```
